lib/_tls_wrap.js

```diff
@@ -625,7 +625,7 @@ function defineHandleReading(socket, handle) {
 
 function onSocketCloseDestroySSL() {
   // Make sure we are not doing it on OpenSSL's stack
-  setImmediate(destroySSL, this);
+  setImmediate(destroySSL.bind(null, this));
```

Comment:
This seems like an unrelated/accidental change? We generally prefer passing arguments explicitly over `.bind()`

Reply:
This is very much on purpose. Please see conversation above.

Reply:
https://github.com/nodejs/node/pull/34103#issuecomment-651040237

Reply:
That should definitely come with a comment here, then

Reply:
What would a good comment be?

Reply:
```suggestion 
  // This should not be using .bind(). However, doing so appears necessary
  // to work around a garbage collection bug:
  // Refs: https://github.com/nodejs/node/pull/34103#issuecomment-651040237
  setImmediate(destroySSL.bind(null, this));
```